backend/app/controllers/spree/admin/products_controller.rb

```diff
@@ -74,6 +74,7 @@ def location_after_save
 
       def load_data
         @tax_categories = Spree::TaxCategory.order(:name)
+        @default_tax_category = @tax_categories.default
```

Comment:
Correct me if I'm wrong, but this will execute an additional SQL query, right? How about selecting the default tax category with `detect(&:is_default)` instead? Small improvement, but no reason why we shouldn't do it.

Reply:
You are correct, _default_ was executing an additional query. I have changed to use _detect_ as you suggested! 
Thanks!